docs/src/pages/style/color.md

```diff
@@ -25,20 +24,46 @@ They’ve been designed to work harmoniously with each other.
 For instance, you can refer to complementary primary and accent colors (for example 'red 500' & 'purple A200'), like so:
 
 ```js
-import { red, purple } from 'material-ui/colors';
+import purple from 'material-ui/colors/purple';
+import red from 'material-ui/colors/red';
 
 const primary = red[500]; // #F44336
 const accent = purple['A200']; // #E040FB
 const accent2 = purple.A200; // #E040FB (alternative method)
 ```
 
-### Full Palette
-{{"demo": "pages/style/Color.js"}}
+## Color tool
+
+Material Design team has built an awesome palette configuration tool: [https://material.io/color](material.io/color).
+It will help you create, share, and apply color palettes to your UI, as well as measure the accessibility level of any color combination.
+
+[![color tool](/static/images/color/colorTool.png)](https://material.io/color/#!/?view.left=0&view.right=0&primary.color=3F51B5&secondary.color=F44336)
+
+The output can be fed directly to `createMuiTheme()` function:
+
+```jsx
+import { createMuiTheme } from 'material-ui/styles';
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      light: '#757ce8',
+      main: '#3f50b5',
+      dark: '#002884',
+      contrastText: '#fff',
+    },
+    secondary: {
+      light: '#ff7961',
+      main: '#f44336',
+      dark: '#ba000d',
+      contrastText: '#000',
+    },
+  },
+});
+```
 
-## Tools
+## Color palette
 
-These are some of the awesome tools that can be used to generate palettes:
+This color palette comprises primary and accent colors that can be used for illustration or to develop your brand colors. They’ve been designed to work harmoniously with each other.
```

Comment:
/accent/secondary/?

Reply:
> primary and accent colors

colors refers to the hue in this context.